lib/components/Navigation/Navigation.module.scss

```diff
@@ -77,8 +77,9 @@ $activeBorderHeight: $layout-nav-item-height - (4*$grid-size);
             width: 2 * $layout-nav-collapsed-width;
             
             :global(.global-nav-item), .top-container, .far-bottom-container {
-                // override the width of the nav-item to match the navbar width:
-                width: $layout-nav-collapsed-width;
+                // override the width of the nav-item to match the navbar width and take into account the 1px border
+                // that is used to delimitate the nav from the main content
+                width: calc(#{$layout-nav-collapsed-width} - 1px);

```

Comment:
We don’t do the -1px when the navbar is expanded, so it seems weird to do it when collapsed. Have you tried replacing the border with outline? That’s what we use for the dotted line in most other components

Reply:
The reason why we need it in collapsed is because when it's not collapsed, the browser just takes care of this calculation, it makes the children be 100% - any border; but on on collapsed, the parent of the items is bigger than it should so the browser cannot make this calculation.

One way we could fix it so that we don't have to calculate it is set `box-sizing: content-box` to the navigation component, so that the size of the items will always be `$layout-nav-collapsed-width` and the border doesn't subtract from them (the downside with this is that the nav itself will not be `$layout-nav-collapsed-width + border-width` on collapsed and `$layout-nav-expanded-width` + border-width on expanded, but this should be okay)